llama_stack/providers/remote/inference/nvidia/openai_utils.py

```diff
@@ -31,17 +30,21 @@
     Choice as OpenAIChoice,
     ChoiceLogprobs as OpenAIChoiceLogprobs,  # same as chat_completion_chunk ChoiceLogprobs
 )
-
 from openai.types.chat.chat_completion_message_tool_call_param import (
     Function as OpenAIFunction,
 )
+from openai.types.completion import Completion as OpenAICompletion
```

Comment:
Is there a reason to not update/use llama_stack/providers/utils/inference/openai_compat.py instead?

Reply:
i would like to find a way to blend these together in the future.

despite the file/func names, at the moment neither is truly openai compat.

when i reviewed the openai_compat.py it appeared to have some implementation specific quirks that i assumed are valid for ollama and tgi, but don't hold for nim (or openai). for instance, the eot/eom/python_tag are not returned from the server, and the finish_reason options are different. likewise, there are some nim specific quirks in this implementation, e.g. nvext params, tools are streamed as a single response message, and response_format isn't a thing for openai /v1/completions.